src/experimental/players/DesktopPauseAfterPlayPlayer/DesktopPauseAfterPlayPlayer.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { DesktopArticleVideoPlayerProps } from 'jwplayer/types';
+import { DesktopArticleVideoPlayerContent } from 'jwplayer/players/DesktopArticleVideoPlayer/DesktopArticleVideoPlayer';
+import usePauseAfterPlays from 'experimental/utils/usePauseAfterPlays';
+import PlayerWrapper from 'jwplayer/players/shared/PlayerWrapper';
+
+const DesktopPauseAfterPlayPlayerContent: React.FC<DesktopArticleVideoPlayerProps> = ({ videoDetails }) => {
+	// TODO: set actual pause after play amount
+	usePauseAfterPlays(1);
```

Comment:
We'll need to change this here to the actual amount specified for the different variants.

Reply:
Yeah, I put a todo just above it when we figure out which to test first.